### ISSUE TITLE
【EESS】New submissions for Fri,  1 Dec 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#923]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Indoor Millimeter Wave Localization using Multiple Self-Supervised Tiny  Neural Networks
 - **Authors:** Anish Shastri, Andres Garcia-Saavedra, Paolo Casari
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG); Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2311.18732
 - **Pdf link:** https://arxiv.org/pdf/2311.18732
 - **Abstract**
 We consider the localization of a mobile millimeter-wave client in a large indoor environment using multilayer perceptron neural networks (NNs). Instead of training and deploying a single deep model, we proceed by choosing among multiple tiny NNs trained in a self-supervised manner. The main challenge then becomes to determine and switch to the best NN among the available ones, as an incorrect NN will fail to localize the client. In order to upkeep the localization accuracy, we propose two switching schemes: one based on a Kalman filter, and one based on the statistical distribution of the training data. We analyze the proposed schemes via simulations, showing that our approach outperforms both geometric localization schemes and the use of a single NN.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Grid-Forming Control of Power Converters: Equivalence Proof through  Simplified Models
 - **Authors:** Aidar Zhetessov
 - **Subjects:** Systems and Control (eess.SY); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2311.17926
 - **Pdf link:** https://arxiv.org/pdf/2311.17926
 - **Abstract**
 This work establishes the equivalence of selected grid-forming control algorithms within the context of simplified theoretical models. Considered algorithms are droop control, Virtual Synchronous Machine (VSM) and matching control. It is shown that nodal and network dynamics under those regulators boil down to the same equations near the selected (trivial) nominal operating point. Finally, some practical insights on each regulator dynamics and an outlook are provided.
### Strategic Workforce Planning in Crowdsourced Delivery with Hybrid Driver  Fleets
 - **Authors:** Julius Luy, Gerhard Hiermann, Maximilian Schiffer
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2311.17935
 - **Pdf link:** https://arxiv.org/pdf/2311.17935
 - **Abstract**
 Nowadays, logistics service providers (LSPs) increasingly consider using a crowdsourced workforce on the last mile to fulfill customers' expectations regarding same-day or on-demand delivery at reduced costs. The crowdsourced workforce's availability is, however, uncertain. Therefore, LSPs often hire additional fixed employees to perform deliveries when the availability of crowdsourced drivers is low. In this context, the reliability versus flexibility trade-off which LSPs face over a longer period, e.g., a year, remains unstudied. Against this background, we jointly study a workforce planning problem that considers fixed drivers (FDs) and the temporal development of the crowdsourced driver (CD) fleet over a long-term time horizon. We consider two types of CDs, gigworkers (GWs) and occasional drivers (ODs). While GWs are not sensitive to the request's destination and typically exhibit high availability, ODs only serve requests whose origin and destination coincide with their own private route's origin and destination. Moreover, to account for time horizon-specific dynamics, we consider stochastic turnover for both FDs and CDs as well as stochastic CD fleet growth. We formulate the resulting workforce planning problem as a Markov decision process (MDP) whose reward function reflects total costs, i.e., wages and operational costs arising from serving demand with FDs and CDs, and solve it via approximate dynamic programming (ADP). Applying our approach to an environment based on real-world demand data from GrubHub, we find that in fleets consisting of FDs and CDs, ADP-based hiring policies can outperform myopic hiring policies by up to 19% in total costs. In the studied setting, we observed that GWs reduce the LSP's total costs more than ODs. When we account for CDs' increased resignation probability when not being matched with enough requests, the amount of required FDs increases.
### Data-Driven Kalman Filter using Maximum Likelihood Optimization
 - **Authors:** Peihu Duan, Tao Liu, Yu Xing, Karl Henrik Johansson
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2311.18096
 - **Pdf link:** https://arxiv.org/pdf/2311.18096
 - **Abstract**
 This paper investigates the state estimation problem for unknown linear systems with process and measurement noise. A novel data-driven Kalman filter (DDKF) that combines model identification with state estimation is developed using pre-collected input-output data and uncertain initial state information of the unknown system. Specifically, the state estimation problem is first formulated as a non-convex maximum likelihood (ML) optimization problem. Then, to reduce the computational complexity, the optimization problem is broken down into a series of sub-problems in a recursive manner. Based on the optimal solutions to the sub-problems, a closed-form DDKF is designed for the unknown system, which can estimate the state of a physically meaningful state-space realization, rather than these up to an unknown similarity transformation. The performance gap between the DDKF and the traditional Kalman filter with accurate system matrices is quantified through a sample complexity bound. In particular, when the number of the pre-collected trajectories tends to infinity, this gap converges to zero. Moreover, the DDKF is used to facilitate data-driven control design. A data-driven linear quadratic Gaussian controller is defined and its closed-loop performance is characterized. Finally, the effectiveness of the theoretical results is illustrated by numerical simulations.
### Audio Prompt Tuning for Universal Sound Separation
 - **Authors:** Yuzhuo Liu, Xubo Liu, Yan Zhao, Yuanyuan Wang, Rui Xia, Pingchuan Tain, Yuxuan Wang
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2311.18399
 - **Pdf link:** https://arxiv.org/pdf/2311.18399
 - **Abstract**
 Universal sound separation (USS) is a task to separate arbitrary sounds from an audio mixture. Existing USS systems are capable of separating arbitrary sources, given a few examples of the target sources as queries. However, separating arbitrary sounds with a single system is challenging, and the robustness is not always guaranteed. In this work, we propose audio prompt tuning (APT), a simple yet effective approach to enhance existing USS systems. Specifically, APT improves the separation performance of specific sources through training a small number of prompt parameters with limited audio samples, while maintaining the generalization of the USS model by keeping its parameters frozen. We evaluate the proposed method on MUSDB18 and ESC-50 datasets. Compared with the baseline model, APT can improve the signal-to-distortion ratio performance by 0.67 dB and 2.06 dB using the full training set of two datasets. Moreover, APT with only 5 audio samples even outperforms the baseline systems utilizing full training data on the ESC-50 dataset, indicating the great potential of few-shot APT.
### A New Old Idea: Beam-Steering Reflectarrays for Efficient Sub-THz  Multiuser MIMO
 - **Authors:** Krishan Kumar Tiwari, Giuseppe Caire
 - **Subjects:** Signal Processing (eess.SP); Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2311.18593
 - **Pdf link:** https://arxiv.org/pdf/2311.18593
 - **Abstract**
 We present a novel, power- & hardware-efficient, multiuser, multibeam RIS (Reflective Intelligent Surface) architecture for multiuser MIMO, especially for very high frequency bands (e.g., high mmWave and sub-THz), where channels are typically sparse in the beamspace and LOS is the dominant component. The key module is formed by an active multiantenna feeder (AMAF) with a small number of active antennas, placed in the near field of a RIS with a much larger number of passive controllable reflecting elements. We propose a pragmatic approach to obtain a steerable beam with high gain and very low sidelobes. Then K independently controlled beams can be achieved by closely stacking K such AMAF-RIS modules. Our analysis includes the mutual interference between the modules and the fact that, due to the delay difference of propagation through the AMAF-RIS structure, the resulting channel matrix is frequency selective even in for pure LOS propagation. We consider a 3D geometry and show that ``beam focusing'' is in fact possible (and much more effective in terms of coverage) also in the far-field, by creating spotbeams with limited footprint both in angle and in range. Our results show that: 1) simple RF beamforming (BF) without computationally expensive baseband multiuser precoding is sufficient to practically eliminate multiuser interference when the users are chosen with sufficient angular/range separation, thanks to the extremely low sidelobe beams; 2) the impact of beam pointing errors with standard deviation as large as 2.5 deg and RIS quantized phase-shifters with quantization bits > 2 is essentially negligible; 3) The proposed architecture is more power efficient & much simpler from a hardware implementation viewpoint than standard active arrays with the same BF performance. We show also that the array gain of the proposed AMAF-RIS structure is linear with the RIS aperture.
### Cancer-Net PCa-Gen: Synthesis of Realistic Prostate Diffusion Weighted  Imaging Data via Anatomic-Conditional Controlled Latent Diffusion
 - **Authors:** Aditya Sridhar, Chi-en Amy Tai, Hayden Gunraj, Yuhao Chen, Alexander Wong
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2311.18612
 - **Pdf link:** https://arxiv.org/pdf/2311.18612
 - **Abstract**
 In Canada, prostate cancer is the most common form of cancer in men and accounted for 20% of new cancer cases for this demographic in 2022. Due to recent successes in leveraging machine learning for clinical decision support, there has been significant interest in the development of deep neural networks for prostate cancer diagnosis, prognosis, and treatment planning using diffusion weighted imaging (DWI) data. A major challenge hindering widespread adoption in clinical use is poor generalization of such networks due to scarcity of large-scale, diverse, balanced prostate imaging datasets for training such networks. In this study, we explore the efficacy of latent diffusion for generating realistic prostate DWI data through the introduction of an anatomic-conditional controlled latent diffusion strategy. To the best of the authors' knowledge, this is the first study to leverage conditioning for synthesis of prostate cancer imaging. Experimental results show that the proposed strategy, which we call Cancer-Net PCa-Gen, enhances synthesis of diverse prostate images through controllable tumour locations and better anatomical and textural fidelity. These crucial features make it well-suited for augmenting real patient data, enabling neural networks to be trained on a more diverse and comprehensive data distribution. The Cancer-Net PCa-Gen framework and sample images have been made publicly available at https://www.kaggle.com/datasets/deetsadi/cancer-net-pca-gen-dataset as a part of a global open-source initiative dedicated to accelerating advancement in machine learning to aid clinicians in the fight against cancer.
### VREM-FL: Mobility-Aware Computation-Scheduling Co-Design for Vehicular  Federated Learning
 - **Authors:** Luca Ballotta, Nicolò Dal Fabbro, Giovanni Perin, Luca Schenato, Michele Rossi, Giuseppe Piro
 - **Subjects:** Systems and Control (eess.SY); Artificial Intelligence (cs.AI); Distributed, Parallel, and Cluster Computing (cs.DC); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2311.18741
 - **Pdf link:** https://arxiv.org/pdf/2311.18741
 - **Abstract**
 Assisted and autonomous driving are rapidly gaining momentum, and will soon become a reality. Among their key enablers, artificial intelligence and machine learning are expected to play a prominent role, also thanks to the massive amount of data that smart vehicles will collect from their onboard sensors. In this domain, federated learning is one of the most effective and promising techniques for training global machine learning models, while preserving data privacy at the vehicles and optimizing communications resource usage. In this work, we propose VREM-FL, a computation-scheduling co-design for vehicular federated learning that leverages mobility of vehicles in conjunction with estimated 5G radio environment maps. VREM-FL jointly optimizes the global model learned at the server while wisely allocating communication resources. This is achieved by orchestrating local computations at the vehicles in conjunction with the transmission of their local model updates in an adaptive and predictive fashion, by exploiting radio channel maps. The proposed algorithm can be tuned to trade model training time for radio resource usage. Experimental results demonstrate the efficacy of utilizing radio maps. VREM-FL outperforms literature benchmarks for both a linear regression model (learning time reduced by 28%) and a deep neural network for a semantic image segmentation task (doubling the number of model updates within the same time window).
### Automated interpretation of congenital heart disease from multi-view  echocardiograms
 - **Authors:** Jing Wang, Xiaofeng Liu, Fangyun Wang, Lin Zheng, Fengqiao Gao, Hanwen Zhang, Xin Zhang, Wanqing Xie, Binbin Wang
 - **Subjects:** Image and Video Processing (eess.IV); Artificial Intelligence (cs.AI); Computer Vision and Pattern Recognition (cs.CV); Multimedia (cs.MM); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2311.18788
 - **Pdf link:** https://arxiv.org/pdf/2311.18788
 - **Abstract**
 Congenital heart disease (CHD) is the most common birth defect and the leading cause of neonate death in China. Clinical diagnosis can be based on the selected 2D key-frames from five views. Limited by the availability of multi-view data, most methods have to rely on the insufficient single view analysis. This study proposes to automatically analyze the multi-view echocardiograms with a practical end-to-end framework. We collect the five-view echocardiograms video records of 1308 subjects (including normal controls, ventricular septal defect (VSD) patients and atrial septal defect (ASD) patients) with both disease labels and standard-view key-frame labels. Depthwise separable convolution-based multi-channel networks are adopted to largely reduce the network parameters. We also approach the imbalanced class problem by augmenting the positive training samples. Our 2D key-frame model can diagnose CHD or negative samples with an accuracy of 95.4\%, and in negative, VSD or ASD classification with an accuracy of 92.3\%. To further alleviate the work of key-frame selection in real-world implementation, we propose an adaptive soft attention scheme to directly explore the raw video data. Four kinds of neural aggregation methods are systematically investigated to fuse the information of an arbitrary number of frames in a video. Moreover, with a view detection module, the system can work without the view records. Our video-based model can diagnose with an accuracy of 93.9\% (binary classification), and 92.1\% (3-class classification) in a collected 2D video testing set, which does not need key-frame selection and view annotation in testing. The detailed ablation study and the interpretability analysis are provided.
